src/pages/jonghyuk/Main/Comment.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useState } from 'react';
+import Newcomment from './Newcomment';
+
+const Comment = () => {
+  const [comment, setComment] = useState('');
+  const hendleComment = event => {
+    setComment(event.target.value);
+  };
+  const [commentArray, setCommentArray] = useState([]);
+  const eventComment = () => {
+    const result = [...commentArray, comment];
+    setCommentArray(result);
+    setComment('');
+  };
+
+  const changeColor = comment.length > 0;
+
+  return (
+    <div className="comment">
+      <div className="like">
+        <img
+          className="commentImg"
+          alt="commentimg"
+          src="images/jonghyuk/feedimg.jpg"
+        />
+        <span className="likeEach">
+          <b>aineworld</b>님 <b>외 10명</b>이 좋아합니다
+        </span>
+      </div>
+
+      <div className="post">
+        <div>
+          <p>
+            <b>canon_mj</b> 위워크에서 진행한 베이킹 클래스...
+          </p>
+        </div>
+        <input className="moreBtn" type="button" value="더 보기" />
+      </div>
+
+      <div className="mention">
+        <div>
+          <b>neceosecius</b> 거봐 좋았잖아~~~~
+        </div>
+        <img className="like" alt="like" src="images/jonghyuk/heart.png" />
+      </div>
+      <div className="lastHour">42분 전</div>
+      <ul className="comment-ul">
+        {commentArray.map(function (a, i) {
+          return <Newcomment key={i} comment={a} />;
+        })}
+      </ul>
+      <div className="makeMention">
+        <input
+          className="writeMention"
+          type="text"
+          placeholder="댓글 달기..."
+          value={comment}
+          onChange={hendleComment}
+        />
+        <input
+          className={changeColor ? 'changeColorBtn' : 'writeBtn'}
```

Comment:
className을 아래와 같은 방식으로 주는 건 어떨까요?
위의 방식처럼 className을 준다면 중복되는 style을 다시 작성해야 할 것 같습니다.

```suggestion
          className={`submitButton ${isFilled ? 'filled' : ''}`}
```

base style은 submitButton을 통해 관리하고
칸이 채워졌을 때에 추가적으로 주고 싶은 style은 .filled 에서 관리하는거죠!